.eslintrc.js

```diff
@@ -19,14 +19,8 @@ const majorMinorRegExp = escapeRegExp( version.replace( /\.\d+$/, '' ) ) + '(\\.
 module.exports = {
 	root: true,
 	extends: [
-		'./eslint/config.js',
-		'plugin:jest/recommended'
-	],
-	env: {
```

Comment:
It turned out `plugin:jest/recommended` includes the related env and plugins config :)